numpy/core/src/umath/cpuid.c

```diff
@@ -57,10 +57,10 @@ npy_cpu_supports(const char * feature)
 {
 #ifdef HAVE___BUILTIN_CPU_SUPPORTS
     if (strcmp(feature, "avx512f") == 0) {
-#if defined(__GNUC__) && (__GNUC__ < 5)
```

Comment:
Do we still need this? I don't see it checked elsewhere.

Reply:
not really, that line existed for the same reason as the bug this PR is fixing. GCC < 5 didn't support `__builtin_cpu_supports("avx512f")`. The preprocessor macro `HAVE___AVX512F_BUILTIN_CPU_SUPPORTS `handles that now.